examples/grpc-bridge/README.md

```diff
@@ -1,2 +1,131 @@
+
+Note that the client container to use is `grpc-bridge_grpc-client_1` and binds to no port
+as it will use the `python` CLI.
+
+```console
+$ docker-compose ps
+             Name                            Command               State                             Ports
+------------------------------------------------------------------------------------------------------------------------------------
+grpc-bridge_grpc-client-proxy_1   /docker-entrypoint.sh /usr ... Up      10000/tcp, 0.0.0.0:9911->9911/tcp, 0.0.0.0:9991->9991/tcp
+grpc-bridge_grpc-client_1         /bin/sh -c tail -f /dev/null     Up
+grpc-bridge_grpc-server-proxy_1   /docker-entrypoint.sh /usr ... Up      10000/tcp, 0.0.0.0:8811->8811/tcp, 0.0.0.0:8881->8881/tcp
+grpc-bridge_grpc-server_1         /bin/sh -c /bin/server           Up      0.0.0.0:8081->8081/tcp
+```
+
+## Use the Client CLI
+
+* Since the containers are running, you can use the client container to interact with the gRPC server through the proxies
+* The client has the methods `set key value` and `get key` to use the in-memory key-value store.
+
+```console
+$ docker-compose exec grpc-client /client/grpc-kv-client.py set foo bar
+setf foo to bar
+```
+
+> NOTE: You could also run docker instead of docker-compose `docker exec -ti grpc-bridge_grpc-client_1 /client/grpc-kv-client.py set foo bar`
+
+* The server will display the gRPC call received by the server, and then the access logs from the proxy for the SET method.
+  * Note that the proxy is propagating the headers of the request
+
+```console
+grpc-server_1        | 2019/11/07 16:33:58 set: foo = bar
+grpc-server-proxy_1  | [2019-11-07T16:33:58.856Z] "POST /kv.KV/Set HTTP/1.1" 200 - 15 7 3 1 "172.24.0.3" "python-requests/2.22.0" "c11cf735-0647-4e67-965c-5b1e362a5532" "grpc" "172.24.0.2:8081"
+grpc-client-proxy_1  | [2019-11-07T16:33:58.855Z] "POST /kv.KV/Set HTTP/1.1" 200 - 15 7 5 3 "172.24.0.3" "python-requests/2.22.0" "c11cf735-0647-4e67-965c-5b1e362a5532" "grpc" "172.24.0.5:8811"
+```
+
+* Getting the value is no different
+
+```console
+$ docker-compose exec grpc-client /client/grpc-kv-client.py get foo
+bar
+```
+
+> NOTE: You could also run docker instead of docker-compose `docker exec -ti grpc-bridge_grpc-client_1 /client/grpc-kv-client.py get foo`
+
+* The logs in the server will show the same for the GET method.
+  * Note that again the request ID is proxied through
+
+```console
+grpc-server_1        | 2019/11/07 16:34:50 get: foo
+grpc-server-proxy_1  | [2019-11-07T16:34:50.456Z] "POST /kv.KV/Get HTTP/1.1" 200 - 10 10 2 1 "172.24.0.3" "python-requests/2.22.0" "727d4dcd-a276-4bb2-b4cc-494ae7119c24" "grpc" "172.24.0.2:8081"
+grpc-client-proxy_1  | [2019-11-07T16:34:50.455Z] "POST /kv.KV/Get HTTP/1.1" 200 - 10 10 3 2 "172.24.0.3" "python-requests/2.22.0" "727d4dcd-a276-4bb2-b4cc-494ae7119c24" "grpc" "172.24.0.5:8811"
+```
+
+# Troubleshooting
+
+* Errors building the `client` or `server` are related for the missing gRPC stubs.
```

Comment:
Related to

Reply:
@snowp awesome! Just fixed it! Are you at `kubecon`?